/rc_control:
  ros__parameters:
    gpiochip: 4
    publish_rate: 50.0
    failsafe_timeout: 1.0
    # Pins: [13, 19, 26, 12, 5, 6]
    # Ch 1-5: Analog Control, Ch 6: Mode Switch
    gpio_pins: [20, 25, 16, 12, 24, 21]